.chloggen/confmap-add-expand-featuregate-2.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: expandconverter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Expanding `$ENV` is no longer supported.  Use `DefaultScheme="env"` or the `otelcol.useStableExpansionRules` feature gate to expand `${}`.
```

Comment:
I find the

>  Expanding `$ENV` is no longer supported.

confusing. If you use this syntax after this change, it will still work, right? It's only if you enable `otelcol.useStableExpansionRules` that this will stop working.

Reply:
That is correct, I'll update this note.